wgsl/index.bs

```diff
@@ -13038,23 +13038,23 @@ Returns the dimensions of a texture, or texture's mip level in texels.
     <tr><td style="width:25%">Parameterization<th>Overload
   </thead>
   <tr algorithm="textureDimensions 1d">
-    <td><var ignore>ST</var> is [=i32=], [=u32=] or [=f32=]<br>
+    <td><var ignore>ST</var> is [=i32=], [=u32=], or [=f32=]<br>
         <var ignore>F</var> is a [=texel format=]<br>
         <var ignore>A</var> is an [=access mode=]<br><br>
         |T| is `texture_1d<ST>` or `texture_storage_1d<F,A>`
     <td><xmp highlight=rust>fn textureDimensions(t: T) -> u32</xmp>
 
   <tr algorithm="textureDimensions 1d level">
-    <td><var ignore>ST</var> is [=i32=], [=u32=] or [=f32=]<br><br>
+    <td><var ignore>ST</var> is [=i32=], [=u32=], or [=f32=]<br><br>
         |T| is `texture_1d<ST>`
 
-        <var ignore>C</var> is [=i32=] or [=u32=]
+        <var ignore>L</var> is [=i32=], or [=u32=]
```

Comment:
nit: I wouldn't bother with the comma here.
Oxford (or serial) comma rule kicks in when three or more elements are in sequence.